### ISSUE TITLE
Discord messages to facebook: getUserInfo error not passed to error reporting

### ISSUE THREAD

**Issue by ajstewart** [miscord/miscord#263]
I had an error where Discord messages were not getting passed onto messenger because of the error below. A Miscord restart solved the problem but the error itself was not passed on to the error reporting channel.

Would it be possible to catch this error?

Have you come across the error before? Seems a common facebook error when googling - wondering if it's avoidable. Thanks!

```
verb discordListener: clean content hello
verb discordListener: username Adam
sill discordListener: message { body: '*Adam*\nhello', attachment: [] }
verb discordListener: threads [ 'snip' ]
verb discordListener Sent message on Messenger
ERR! getUserInfo { __ar: 1,
ERR! getUserInfo   error: 1357004,
ERR! getUserInfo   errorSummary: 'Sorry, something went wrong',
ERR! getUserInfo   errorDescription: 'Please try closing and re-opening your browser window.',
ERR! getUserInfo   payload: null,
ERR! getUserInfo   bootloadable: {},
ERR! getUserInfo   ixData: {},
ERR! getUserInfo   gkxData: {},
ERR! getUserInfo   lid: 'snip' }
```

**Comment by ajstewart** [comment]
I see that you mentioned on the Discord that you could make Miscord crash on this error - might be a useful addition (if the error can be reported back first).

**Comment by Bjornskjald** [comment]
isn't it crashing already?

**Comment by ajstewart** [comment]
No, I had to manually restart miscord (via pm2) to get it working again. As it was, every discord message received was returning the above error but not crashing miscord, so nothing was being sent that way. Facebook -> Discord was working ok.

**Comment by Bjornskjald** [comment]
I think it was fixed in #277, closing
